app/assets/javascripts/pages/static_pages/about.js

```diff
@@ -32,9 +42,12 @@
       }
     },
 
-    openModal: function (slug) {
+    openModal: function (data) {
+      console.log(data);
       new App.Component.ModalTeam({
-        slug: slug
+        showTitle: false,
```

Comment:
Instead of doing that, you could add the property to the `defaults` object of `App.Component.ModalTeam`.